pkgs/by-name/cw/cwtch-ui/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  cwtch,
+  fetchgit,
+  flutter,
+  lib,
+  tor,
+  zenity,
+}:
+let
+  runtimeBinDependencies = [
+    tor
+    zenity
+  ];
+in
+flutter.buildFlutterApplication rec {
+  pname = "cwtch-ui";
+  version = "1.15.1";
+  src = fetchgit {
```

Comment:
Why not `fetchFromGitea`?

Reply:
I tried that initially but it didn't work. The conclusion I drew at that time was that this particular Gitea instance has archive downloads disabled, so you have to speak the Git protocol to get the code.

Reply:
Would be a good idea to document that then